cmd/kube-vip.go

```diff
@@ -230,12 +234,17 @@ var kubeVipManager = &cobra.Command{
 		if initConfig.EnableWireguard {
 			mode = "Wireguard"
 		}
+
+		if initConfig.EnableRoutingTable {
+			mode = "Routing Table"
```

Comment:
Are those modes (wireguard, arp, bgp, routing table) mutually exclusive? Would it make sense to have a single  `--mode={arp,bgp,wireguard,routingtable}` cli argument then?

Reply:
It probably would, but we need to maintain backwards compatibility :(